src/private/conversations.ts

```diff
@@ -0,0 +1,66 @@
+import { ensureInitialized, sendMessageRequest } from "../internal/internalAPIs";
+import { GlobalVars } from "../internal/globalVars";
+import { frameContexts } from "../internal/constants";
+import { StartConversationRequest, ShowConversationRequest } from "../public/interfaces";
+
+/**
+ * Namespace to interact with the conversational subEntities inside the tab
+ */
+export namespace conversations {
+
+  /**
+  * @private
+  * Hide from docs
+  * --------------
+  * Allows the user to start a conversation with each subentity inside a tab
+  * @param startConversationRequest Callback containing the conversation Id and if the tab pane was closed
+  */
+  export function startConversation(
+    startConversationRequest: StartConversationRequest
+  ): void {
+    ensureInitialized(frameContexts.content);
+    const messageId = sendMessageRequest(GlobalVars.parentWindow, "conversations.startConversation", [{
+      title: startConversationRequest.title,
+      subEntityId: startConversationRequest.subEntityId
+    }]);
+    GlobalVars.callbacks[messageId] = (conversationId?: string, reason?: string) => {
+      if (conversationId) {
+        startConversationRequest.onStartConversation(conversationId);
+      } else {
+        startConversationRequest.onCloseConversation(reason);
+      }
+    };
+  }
+
+  /**
+  * @private
+  * Hide from docs
+  * --------------
+  * Allows the user to show the conversation in the right pane
+  * @param showConversationRequest Callback containing if the tab pane was closed
+  */
+  export function showConversation(
```

Comment:
Let's get some unit tests added for all of this new functionality once we've finalized the interface.